src/System.Private.Xml/tests/XmlTextReader/System.Xml.XmlTextReader.Tests/XmlTextReaderAttributeTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Xml.Tests
+{
+    public class XmlTextReaderAttributeTests
+    {
+        [Fact]
+        public void XmlTextReaderGetAttributeWithIndexTest()
```

Comment:
I think this should be couple of more test cases:
- negative index
- 0 (as is), 1 (add 1 more attribute), 2 (out of bounds)

Reply:
also you can remove the prefix "XmlTextReader" from the test case name as it is already contained in the class name

Reply:
also one more test case where value is not there at all